packages/client/components/ReflectionCard/ReflectionCard.tsx

```diff
@@ -317,18 +317,18 @@ const ReflectionCard = (props: Props) => {
       onMouseLeave={() => setIsHovering(false)}
       showDragHintAnimation={showDragHintAnimation}
       ref={reflectionDivRef}
-      className='py-2'
+      className='py-3'
     >
       <ColorBadge phaseType={phaseType as NewMeetingPhaseTypeEnum} reflection={reflection} />
 
       <div
         ref={scrollRef}
-        className={cn('relative w-full overflow-auto text-sm leading-4 text-slate-700')}
+        className={cn('relative w-full overflow-auto text-sm leading-5 text-slate-700')}
       >
         <TipTapEditor
           className={cn(
-            'flex min-h-4 w-full px-4',
-            isClipped ? 'max-h-8' : 'max-h-28',
+            'flex min-h-6 w-full px-4',
+            phaseType === 'discuss' ? 'min-h-0' : isClipped ? 'max-h-6' : 'max-h-41',
```

Comment:
+1 this seems like an odd ternary-- either it has a min-h OR a max-h? would it be better to say `phaseType === 'discuss' && 'min-h-0' ` as a separate line item?

Reply:
Actually, what I should do is not set anything if it’s the discuss phase, otherwise set a max-height based on if `isClipped` or not. I want to allow the full size of the cards during discuss. The only change I want to make here is to set it `undefined` not `min-h-0` which is inaccurate, unnecessary. Good eye!